### ISSUE TITLE
Can't quite run from source

### ISSUE THREAD

**Issue by felix91gr** [aichingm/lonewolf#4]
Hi. I've been using the app for a couple of days on Ubuntu, and I love it. I wanted to build a tauri `.exe` for a friend whose working setup is in Windows, so that he may run it locally as well, but I haven't been able to run the dev environment (nor any of the build scripts). I think I'm messing up in something really trivial. This is what I've done:

0. Install `podman` version `4.6.2`.
1. Run `make image-dev`, which as far as I can tell, completes without any major issues.
2. Attempt to run `make dev-web`. It starts, but fails with the following output:

```console
/bin/sh: 1: [: 1: unexpected operator
/bin/sh: 1: [: 1: unexpected operator
podman run --rm -it -v .:/app -w /app/lonewolf-web lonewolf:dev bash -c 'convert -background transparent src/assets/icon.svg -resize 64x64 -format ico public/favicon.ico'
/bin/sh: 1: [: 1: unexpected operator
podman run --rm -it -v .:/app -w /app/lonewolf-web lonewolf:dev bash -c 'convert -background transparent src/assets/lonewolf.svg -resize 512x512 -format png public/lonewolf.png'
/bin/sh: 1: [: 1: unexpected operator
podman run --rm -it -v .:/app -w /app/lonewolf-web lonewolf:dev bash -c 'convert -background transparent src/assets/icon.svg -resize 128x128 -format png public/icon.png'
/bin/sh: 1: [: 1: unexpected operator
podman run --rm -it -p 5173:5173 -v .:/app -w /app/lonewolf-web lonewolf:dev bash -c 'npm run dev -- --host'

> lonewolf@0.0.0 dev
> vite --host

sh: line 1: vite: command not found
make: *** [Makefile:81: dev-web] Error 127

```

It's been a while since I've used Makefiles, and this is my first time using one for containerized environments such as with `podman`. So I suspect I must be missing something real dumb, but I'm not sure what.

**Comment by aichingm** [comment]
It seams to me that you have to run `make npm-install` before running `make dev-web`.

**Comment by felix91gr** [comment]
That did it! Thank you @aichingm

Hm. This should probably be added to the README, right?

**Comment by aichingm** [comment]
@felix91gr The build system is currently not in a state in which I would like it to be. Maybe I get to rework it over the weekend. No promises tho!

**Comment by felix91gr** [comment]
> @felix91gr The build system is currently not in a state in which I would like it to be. Maybe I get to rework it over the weekend. No promises tho!

Ohh, if you're planning on reworking it... could I ask for a direct command to compile it for Windows? :smile_cat: 

Since as far as I can tell, Tauri should allow it to be packaged for any OS for which Rust code can be compiled, but I tried modifying the Makefile to make a windows executable and ended up making somewhat of a chimeric `.deb` instead :smiling_face_with_tear:

**Comment by aichingm** [comment]
Hi @felix91gr I split up the makefiles for both of the projects (web/tauri) which can now be run without a containerization. reading the make files should now be easier. 

However there there are some things which need to be done before `npm run tauri build` can work on windows:
* convert the lonewolf logo from svg to png 
* download the icons used in the application from api.iconify.design and convert them to a .ts file

Since I don't develop on/for windows I have no real idea on how to do this. If this is something you would be interested in doing, I would be open to any and all suggestions on how to include it in to the codebase.

**Comment by felix91gr** [comment]
Okis, I shall open a new issue specifically to address this and track progress on it!

**Comment by felix91gr** [comment]
Hm. Now that it's not containerized, I have no idea of what do I need to install to make things work. Do you have a list of the system dependencies I need? I can make a PR to add them to the README if you'd like. I'm working on Ubuntu

**Comment by felix91gr** [comment]
(CC @aichingm since after an issue gets closed I'm not 100% sure if you get new comment notifications)
